src/components/Text/Text.tsx

```
@@ -0,0 +1,179 @@
+      weightClasses = 'font-semibold';
+      break;
+    case 'bold':
+      weightClasses = 'font-bold';
+      break;
+    case 'extrabold':
+      weightClasses = 'font-extrabold';
+      break;
+    case 'black':
+      weightClasses = 'font-black';
+      break;
+    case 'normal':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)